src/main/java/core/basesyntax/dao/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReaderImpl implements CsvFileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        List<String> dataFromFile = new ArrayList<>();
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + filePath + e);
```

Comment:
Consider adding a space or separator between the file path and the exception message in the `RuntimeException` to improve readability. For example, use `"Can't read data from file: " + filePath + ". " + e`.